ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -130,6 +130,16 @@ public static long get_current_epoch(BeaconState state) {
     return slot_to_epoch(state.getSlot());
   }
 
+  public static long get_previous_epoch(BeaconState state) {
+    if (get_current_epoch(state) > slot_to_epoch(Constants.GENESIS_SLOT))
```

Comment:
While this sort of logic is more concise, I think for clarity a standard `if()...else` would make sense here. It's just easier to read through and quickly understand. It took a second to figure out the default return case was when we're still on the first epoch.

Reply:
Fixing this to conform with the comments